app/presenters/export/csv/outgoing_trust_presenter_module.rb

```diff
@@ -16,4 +16,18 @@ def outgoing_trust_name
 
     @project.outgoing_trust.name
   end
+
+  def outgoing_trust_main_contact_name
+    contact = @project.outgoing_trust_main_contact_id
+    return unless @project.outgoing_trust_main_contact_id.present?
```

Comment:
You can swap these two lines around - no need to assign the ID to `contact` if it doesn't exist.